manifests/input.yaml

```
@@ -111,6 +116,11 @@ work/INPUT/salt_sfc_restore.nc:
   hashes:
     binhash: a5bf255da3f43e764cc25469bfe816e3
     md5: 6e37313d88f867af9f1d9f656ab633c9
+work/INPUT/tideamp.nc:
+  fullpath: /g/data/vk83/prerelease/configurations/inputs/access-om3/mom/tidal_external_files/global.25km/2025.05.15/tideamp.nc
+  hashes:
+    binhash: 39ad0015c5a8aed0f4f37e3c81cfa8ae
+    md5: b181ad94293dcd9d794addf9847ddc3e
```

These hashes have changed. In the future, rather than modify input files in place, please just create a new version
```suggestion
    binhash: 1e061e784b1302304c67a1d6c23360a7
    md5: dc4b570589028ce3d09d55d64a7153f5
```